File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/03_compito_java_20150216/Prodotto.py
class Prodotto:
    def __init__(self, tipo, codice, titolo, autore, prezzo):
        self._tipo = tipo
        self._codice = codice
        self._titolo = titolo
        self._autore = autore
        self._prezzo = prezzo

    def __str__(self):
        return self._tipo + "\t" + str(self._codice) + "\t" + self._titolo + "\t" + self._autore + "\t" + str(self._prezzo)

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/03_compito_java_2015_02_16/Prodotto.java
public class Prodotto {
    protected int codice;
    protected String titolo;
    protected String autori;
    protected int prezzo;

    public Prodotto(int codice, String titolo, String autori, int prezzo) {
        this.codice = codice;
        this.titolo = titolo;
        this.autori = autori;
        this.prezzo = prezzo;
    }

    public int getPrezzo() {
        return prezzo;
    }
}